### ISSUE TITLE
Add an Accessibility Statement to the footer

### ISSUE THREAD

**Issue by mgifford** [wet-boew/wet-boew#8548]
It is a best practice to include an accessibility statement in the footer of your webpage indicating what efforts have been done to make your site accessible. The [UK will be requiring it on new sites](https://www.gov.uk/guidance/accessibility-requirements-for-public-sector-websites-and-apps#accessibility-statements), It is a best practice that has [come from the USA](https://www.lflegal.com/2013/02/access-info-pages/), but that should also be considered as a part of the WxT framework.

I think it could be useful for the GC to adopt.

**Comment by juleskuehn** [comment]
EU model accessibility statement (see Annex C) and discussion: https://ec.europa.eu/info/law/better-regulation/initiatives/ares-2018-2604172_en

**Comment by duboisp** [comment]
fyi - @patlaj

**Comment by patlaj** [comment]
This is a great idea and seems very overdue... I'll see how to get this moving.

**Comment by mgifford** [comment]
I've tried to define elements that would be part of a best practice for government https://github.com/accessibility/Accessibility-Statement

I am pretty sure that we can do better than some government departments who define their commitment to accessibility as:

"The Government of Canada is committed to achieving a high standard of accessibility as defined in the Standard on Web Accessibility and the Standard on Optimizing Websites and Applications for Mobile Devices. In the event of difficulty using our Web pages, applications or device-based mobile applications, please contact us for assistance or to obtain alternative formats such as regular print, Braille or another appropriate format."

Saying 'we follow TBS guidelines' isn't really giving much value to anyone.

**Comment by patlaj** [comment]
Yeah we can definitely expand those to describe the meaningful parts of each. The current reality is that all GC sites should meet WCAG 2.0 AA and the writing guidelines in the Canada.ca Style Guide. That's probably the core of what would be most useful to include.

**Comment by mgifford** [comment]
True, but it should go beyond that. Are ARIA landmarks used? What should users do if they discover a problem? How can I personalize this site for my disability?  Is there a roadmap for including features like darkmode? Has it been tested for VoiceOver & other popular assistive technology?

**Comment by patlaj** [comment]
Yep - I wonder if we could start building it out here, on a wiki page maybe?

Also, does it make sense to have separate notices based on the context? Some things should be universal (complying to guidance), but others will be contingent on where it came from (how much testing they did and with what).

**Comment by mgifford** [comment]
We could create a page here or even up here https://github.com/accessibility/Accessibility-Statement/wiki

I think that the pages are going to have to differ from site to site. Some things will be the same, but not all sites are evergreen. Having a short paragraph of policy be consist across government would be a bit useful. Then allow other content to describe how the site is set up in more context..  

I like the idea.

**Comment by patlaj** [comment]
Yeah I think you're right... one thing is that "site" is hard to define now ... there are plenty of pages that look like standard Canada.ca, but are on different servers. Sometimes they have different domains. For example, https://weather.gc.ca/canada_e.html . Technically that's a different site, but there's no reason for visitors to think it is. It's all Canada.ca really. And of course their testing regimen is going to be different from the home page and top-level pages of Canada.ca.

We'd just have to figure out how different pages could include their own additions to the central statement (which could focus on the overarching requirements and WET testing).

**Comment by patlaj** [comment]
It seems like the best solution might be a generic statement in the footer, that can then be customized by the specific content owner (if appropriate). It could work like the "Contact us" link currently in the main footer, which points to a generic contact page but can point to a context-specific one if it exists. 

Then we'd just need to make sure the context-specific statements point to the global one in a standardized way to it's easy to get to both.

**Comment by mgifford** [comment]
The problem with generalized generic statements is that they just become boiler plate documents that nobody reads or understands.  How do we make them relevant and regularly updated so that it provides a useful guide to users?

**Comment by cfarquharson** [comment]
@mgifford I see you have a sample statement but it's 2 years old.  Let's connect as I'm working on this for CRA.  I believe you called me in the past.  Can you call me again?

**Comment by mgifford** [comment]
I'm just checking in with this to see how it's going. It would be nice to know that there is a model up somewhere (even if it isn't published yet to a live site).

**Comment by mgifford** [comment]
Just nudging this issue ahead with a recent Medium post I wrote https://medium.com/civicactions/whitehouse-gov-makes-an-accessibility-statement-5de37580209

**Comment by shawnthompson** [comment]
hey @cfarquharson long time no message!

Why just for CRA? why not push to have it on Canada.ca? 

Let's get on this with Peter Smith and the Canada.ca design team. I'll see if AWG will support it.

**Comment by mgifford** [comment]
Just checking in on this again. It does really seem like one of the easiest ways to determine how effectively governments are meeting the requirements of the Accessible Canada Act. Making it easy to hear from people with disabilities is key.
